rules/do-you-use-ai-for-ui/rule.md

```diff
@@ -0,0 +1,45 @@
+---
+type: rule
+archivedreason: 
+title: Do You Use AI in UI Design?
+guid: b6849044-e11e-4a25-95a1-457493ee9eab
+uri: ai-in-ui-design
+created: 2024-01-09T19:11:41.0000000Z
+authors:
+- title: Luke Cook
+  url: https://ssw.com.au/people/luke-cook
+- title: Gordon Beeming
+  url: https://ssw.com.au/people/gordon-beeming
+- title: Joseph Fernandez
+  url: https://ssw.com.au/people/joseph-fernandez
+- title: William Liebenberg
+  url: https://ssw.com.au/people/william-liebenberg
+related: []
+redirects: []
+
+---
+
+
+When a web developer is handed design images, it can be a drag to spend the next few hours (or days/weeks) building the baseline components, HTML, and CSS before getting to the fun stuff.
+
+Thankfully, AI-assisted tools are beginning to help with the more mundane aspects of this process, and sites such as [v0.dev](https://v0.dev) can produce React components you can drop into your project to give you a huge headstart on the work!
+
+`youtube: https://www.youtube.com/watch?v=By9wCB9IZp0`
```

Comment:
Hey @lukecookssw 

- [ ] Please add a caption as per https://www.ssw.com.au/rules/add-useful-and-concise-figure-captions/#tip-5-add-the-titles--length-on-video-captions

Reply:
Done